scout/server/blueprints/institutes/templates/overview/institute_settings.html

```diff
@@ -31,11 +31,12 @@
   <div class="row" id="body-row"> <!--sidebar and main container are on the same row-->
     {{ institute_actionbar(institute) }} <!-- This is the sidebar -->
   <div class="col">
-    {{ institute_settings(form, institute, current_user, loqus_instances) }}
+    <div>
```

Comment:
Not that it matters, but why the extra div?

Reply:
Naa, they're just extra, I've removed them, thanks!